src/CoreWCF.Http/tests/Authorization/AuthorizationTests.NotSupported.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ServiceModel.Security;
+using System.Threading.Tasks;
+using CoreWCF.Channels;
+using CoreWCF.Configuration;
+using CoreWCF.Description;
+using CoreWCF.IdentityModel.Claims;
+using CoreWCF.IdentityModel.Policy;
+using Helpers;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace CoreWCF.Http.Tests.Authorization;
+
+public partial class AuthorizationTests
+{
+
+    public static IEnumerable<object[]> Get_Authorization_Features_Are_Mutually_Exclusive_TestVariations()
+    {
+        yield return new object[]
+        {
+            typeof(ThrowingStartupWithServiceAuthorizationManager<SinglePolicyOnOperationContractSecuredService>)
+        };
+        yield return new object[]
+        {
+            typeof(ThrowingStartupWithAuthorizationPolicies<SinglePolicyOnOperationContractSecuredService>)
+        };
+    }
+
+    [Theory]
+    [MemberData(nameof(Get_Authorization_Features_Are_Mutually_Exclusive_TestVariations))]
+    public void Authorization_Features_Are_Mutually_Exclusive_Test(Type startupType)
+    {
+        using IWebHost host = ServiceHelper.CreateWebHostBuilder(_output, startupType).Build();
+        var exception = Assert.Throws<NotSupportedException>(() =>
+        {
+            host.Start();
+        });
```

Comment:
This will leak an open host if there's a bug which stops it from throwing. Do this to ensure that doesn't happen:
```c#
using(host)
{
    host.Start();
}
```
This will guarantee the host is disposed regardless of the outcome of Start().